src/spans/awsSpan.ts

```diff
@@ -254,11 +271,18 @@ export const getHttpInfo = (requestData, responseData): HttpInfo => {
     } else {
       request.headers = payloadStringify(request.headers, sizeLimit);
       request.body = payloadStringify(decodeHttpBody(request.body, isError), sizeLimit);
-
+      if (response.body) {
+        if (isJsonContent(response.body, response.headers) && isContainingSecrets(response.body)) {
+          if (!(response.body.length < sizeLimit)) {
+            response.body = untruncateJson(response.body);
```

Comment:
does `untruncateJson` return an object or a string?
This is a waste to change it from object to string and back 🤔

Reply:
a string